webpack.config.js

```diff
@@ -31,6 +31,11 @@ const coreConfig = (env = {}) => ({
       }],
     }),
   ],
+  resolve: {
+    fallback: {
+      util: require.resolve('util/'),
```

Comment:
Can you explain why this is needed?

Reply:
I had an error: "Module not found: Error: Can't resolve 'util' in webpack"

Solution: https://stackoverflow.com/questions/64402821/module-not-found-error-cant-resolve-util-in-webpack